Backend/app/rider-platform/rider-app/Main/src/Storage/Beam/Merchant.hs

```diff
@@ -199,6 +201,7 @@ merchantTMod =
       signatureExpiry = B.fieldNamed "signature_expiry",
       updatedAt = B.fieldNamed "updated_at",
       createdAt = B.fieldNamed "created_at",
+      timeDiffFromUtc = B.fieldNamed "time_diff_from_utc",
```

Comment:
Why is this added here when you could get it from `transporterConfig` table?

Reply:
hemant told me to add this in merchant table as it was in driver side

Reply:
This is rider @0utkarsh . We don't have transporterConfig in rider app.